client/src/components/application/NonlinearStepper/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Container, ThemeProvider } from '@material-ui/core';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepButton from '@material-ui/core/StepButton';
+import ProInfos from './ProInfos';
+import GeneralInfos from './GeneralInfo';
+import { Theme } from '../../common/Typography/style';
+import Button from '../../common/Button';
+import nLinearStepperValidation from '../../../utils/application/nLinearStepperValidation';
+import useStyles from './style';
+
+export default function HorizontalNonLinearStepper() {
+  const classes = useStyles();
+  const [activeStep, setActiveStep] = useState(0);
+  const [formValues, setFormValues] = useState({
+    gender: '',
+    fullName: '',
+    MobNumber: '',
+    age: '',
+    adress: '',
+    eConfident: '',
+    eUnderstand: '',
+    currentEmploy: '',
+    jobTitle: '',
+  });
+  const handleFormInput = (e) => {
+    setFormValues({
+      ...formValues,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const steps = ['General Information', 'Professional Information'];
```

Comment:
add it before the functions, like groups , variables firstly then functions under it